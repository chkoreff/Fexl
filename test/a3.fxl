# This script ensures that all the expected functions are defined, simply by
# mentioning all of them.

print "SUCCESS:  All the expected symbols are defined.";nl;

F  # The F prevents evaluating the following expression.
(
<
<=
=
==
>
>=
-
!=
?
/
*
+
all
and
append
argc
args
argv
atomic
base_path
C
char_get
char_put
choose
compare
default
defined
do
double_add
double_compare
double_div
double_eq
double_ge
double_gt
double_le
double_long
double_lt
double_max
double_min
double_mul
double_ne
double_string
double_sub
double_type
end
envc
envp
envs
exit
F
fgetc
file_string
filter
find
fold
fopen
fputc
fwrite
get_max_bytes
get_max_steps
halt
I
is_apply
is_double
is_long
is_name
is_string
item
L
list_and
list_at
list_entry
list_long
list_or
list_prefix
long_add
long_char
long_compare
long_div
long_double
long_eq
long_ge
long_gt
long_le
long_lt
long_max
long_min
long_mul
long_ne
long_string
long_sub
long_type
map
max
min
name_string
nl
no
not
or
pair
parse
parse_file
print
put_max_bytes
put_max_steps
query
quote_string
R
rand
resolve
reverse
ru_utime
S
show
some
sort
string_append
string_at
string_common
string_compare
string_double
string_eq
string_ge
string_gt
string_index
string_le
string_len
string_long
string_lt
string_max
string_min
string_name
string_ne
string_put
string_slice
string_stderr
string_type
T
to_string
type_eq
type_named
type_of
undefined
value_put
value_string
warn
Y
yes
)
