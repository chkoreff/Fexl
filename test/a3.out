SUCCESS:  All the expected symbols are defined.
